app/helpers/format-price.js

```diff
@@ -0,0 +1,13 @@
+import { helper } from '@ember/component/helper';
+
+export function formatPrice(params/*, hash*/) {
+	let price = parseInt(params[0]);
+	if(price == 0){
```

Comment:
Just check for `NaN` here.

Reply:
sure, what should i return in case for NaN?

Reply:
If there is no price, consider that as 0

Reply:
And thus free